agent/04_agent_configure.sh

```diff
@@ -268,6 +268,7 @@ else
     ip=${AGENT_NODES_IPSV6[0]}
   fi
   configure_dnsmasq ${ip} ""
+  echo "${ip}" >> ${OCP_DIR}/node0-ip
```

Comment:
Any reason we are appending (`>>`) instead of replacing (`>`) here?

Reply:
Agree `>` is better here else I have to make sure only the first item from the node0-ip file is used because of the unnecessary appending with `>>` and had to do `ip=$(head -n 1 ${OCP_DIR}/node0-ip)`. With `>` we can rely on `ip=$(<${OCP_DIR}/node0-ip)`. Thank you.